app/(candidate)/dashboard/shared/ProUpgradeModal.js

```diff
@@ -7,21 +7,23 @@ import Body2 from '@shared/typography/Body2'
 import Button from '@shared/buttons/Button'
 import { trackEvent, EVENTS } from 'helpers/fullStoryHelper'
 
-const VIABILITY_SCORE_THRESHOLD = 2
+export const VIABILITY_SCORE_THRESHOLD = 2
 const LOCAL_STORAGE_KEY = 'proUpgradeModalDismissedSession'
-const VARIANTS = {
+
+export const VARIANTS = {
   First: 'First',
   Second_NonViable: 'Second_NonViable',
   Second_Viable: 'Second_Viable',
   Third: 'Third',
 }
+
 const SESSION_TRIGGERS = {
   First: 3,
   Second: 8,
   Third: 12,
 }
 
-export default function ProUpgradeModal({ campaign, user }) {
+export function ProUpgradePrompt({ campaign, user }) {
```

Comment:
We should also rename the file to match the component name.

Reply:
Oh, I see now that the original component is still exported from this file. 

We should probably have a separate file for this new component.